models/CoronSegmentator/scripts/coronaryArtery_segmentation/coronaryArtery_seg.py

```
@@ -0,0 +1,608 @@
+        with concurrent.futures.ThreadPoolExecutor(
+            max_workers=min(int(os.cpu_count() * 0.75), len(nii_files))
+        ) as executor:
+            futures = [
+                executor.submit(process_file, nii_file) for nii_file in nii_files
+            ]
+            # Wait for all tasks to complete and handle exceptions
+            for future in concurrent.futures.as_completed(futures):
+                try:
+                    future.result(timeout=300)
+                except TimeoutError as te:
+                    logger.error(f"TimeoutError: {str(te)}")
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)